.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
 #
 before_script:
   - cd $TRAVIS_BUILD_DIR/..
-  - git clone --branch master --depth=1 https://github.com/xbmc/xbmc.git
+  - git clone --branch change-vfs2 --depth=1 https://github.com/AlwinEsch/kodi.git xbmc
```

Comment:
you might want to revert this ;)

Reply:
Thats why "add **temporary** own repo for build tests" :smile: